docs/quickstart.rst

```diff
@@ -270,13 +269,13 @@ for a passphrase to encrypt/decrypt the keyfile / repokey.
 A passphrase should be a single line of text, a trailing linefeed will be
 stripped.
 
-For your own safety, you maybe want to avoid empty passphrases as well
+For security, you should avoid empty passphrases and 
 extremely long passphrase (much more than 256 bits of entropy).
```

Comment:
Does borg intend to manage the physical safety of its user, or the integrity and recover-ability of users data? (we do not digress into what may happen to a user who loses important data. Let's keep it simple...)

I will rewrite. Do you have statistics that show that longer passwords are more likely to lose data?

Reply:
It's about recoverability of users data, ofc.

I don't have statistics, but I know from my work that many users struggle a lot with passwords. Also, I can imagine that some tools and password managers might truncate very long passphrases. Considering it is also pointless to go way beyond 256bits of entropy, I think it is valid to advise against extremely long passphrases.

Reply:
Well, entropy. Is the system and its components capable of handling more than 256 bits without truncation? Getting sidetracked with suppositions and generalizations might instill doubt in borg.

I will think something up for the 256 bit entropy pending your answer.

Reply:
borg can deal with any length, but it will be just fed into a key derivation function that creates a 256bit AES key from it.

Reply:
So the important part: is the output result of the derivation function the same if we truncate the inputs, even though the AES key will be 256 bit? Each of your answers seems to hinge on something else - so what I am trying to elicit is that no functionality suffers due to longer passwords, only that the likelihood of user mistakes increases.

Reply:
The KDF will generate different 256bit outputs for different inputs (except rare collisions), but it will never output more than 256bits.

The risk here is not in borg, the risk is things going wrong externally (human errors, external limitations).

Reply:
Let's remove this bit - we want a quickstart, not a rationalization about all potential problems along the way. :)

Perhaps we can just include a link to the xkcd correct-horse-battery-staple? 😄

Reply:
ok, remove it and add the xkcd.

3rd xkcd link in our docs. :-)